brain/circuit.cpp

```diff
@@ -49,17 +50,18 @@ URIs _getMorphologyURIs(const Circuit::Impl& circuit, const GIDSet& gids)
 
 Matrix4fs _getTransforms(const Circuit::Impl& circuit, const GIDSet& gids)
 {
-    const Vector3fs& positions = circuit.getPositions(gids);
-    const Quaternionfs& rotations = circuit.getRotations(gids);
+    const Vector3fs positions = circuit.getPositions(gids);
+    const Quaternionfs rotations = circuit.getRotations(gids);
     if (positions.size() != rotations.size())
         throw std::runtime_error(
             "Positions not equal rotations for given GIDs");
 
     Matrix4fs transforms(positions.size());
 
-#pragma omp parallel for
+    #pragma omp parallel for
     for (size_t i = 0; i < positions.size(); ++i)
-        transforms[i] = Matrix4f(rotations[i], positions[i]);
+        transforms[i] = glm::translate(glm::mat4(1.f), positions[i]) * glm::mat4_cast(rotations[i]);
```

Comment:
Are you sure ? This shit can be tricky....

Reply:
I tested both ways. The rotation then translate was wrong. This one yields the same result as with vmmlib